
<launch>
    <!-- VIO through flvis -->
    <!-- <include file="$(find flvis)/launch/flvis_d455_depth.launch"/> -->

    <!-- ROS Nodelet manager -->
    <node pkg="nodelet" type="nodelet" name="alan_master" args="manager" output="screen">
            <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/alan_pose_estimation.yaml" />
            
    </node> 


    <!-- ArucoNodelet -->
    <!-- <node pkg="nodelet" type="nodelet" name="aruco" args="load alan/ArucoNodelet alan_master" output="screen">
    </node> -->
    <!-- <nodec -->

    <!-- CnnNodelet -->
    <!-- <node pkg="nodelet" type="nodelet" name="cnn" args="load alan/CnnNodelet alan_master" output="screen">
        <param name="/weightpath"    type="string" value="$(find alan_state_estimation)/launch/configs/yolo/uav_new.weights"/>
        <param name="/cfgpath"       type="string" value="$(find alan_state_estimation)/launch/configs/yolo/uav_new.cfg    "/>
        <param name="/classnamepath" type="string" value="$(find alan_state_estimation)/launch/configs/yolo/uav.names      "/>
        <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/CNN_SUBPUB_topics.yaml" />
    </node> -->

    <!-- LedNodelet -->
    <node pkg="nodelet" type="nodelet" name="led" args="load alan/LedNodelet alan_master" output="screen">
        <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/LED_SUBPUB_topics.yaml" />
    </node> 


    <!-- MsgSyncNodelet -->
    <node pkg="nodelet" type="nodelet" name="msgsync" args="load alan/MsgSyncNodelet alan_master" output="screen">
        <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/MSGSYNC_SUBPUB_topics.yaml" />
    </node> 


    <!-- <node name="virtual_car" pkg="alan_state_estimation" type="virtual_car" output="screen">
        <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/VIRTUAL_CAR_SUBPUB_topics.yaml" />
        <param name="/pathcsv" type="string" value="$(find alan_state_estimation)/launch/configs/virtual_car_path/ugv_still_path.csv"/>
        <param name="/max_vel_lin" type="double" value="0.75"/>
        <param name="/max_vel_ang" type="double" value="15"/>
        <param name="/pub_freq" type="int" value="200"/>
    </node> -->


</launch>
